index.html

```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
```

Comment:
```So yeah here we go
  </tr>
```

Reply:
Wait how do you commit suggestion like where????
```suggestion
  Wait how do you commit suggestion like where????
```